CMakeLists.txt

```diff
@@ -158,18 +188,26 @@ if(CUDA_FOUND)
     include_directories(${NvToolExt_INCLUDE_DIR})
     link_libraries(${NvToolExt_LIBRARIES})
 
-    get_third_party(cub)
-    set(CUB_ROOT_DIR "${CMAKE_CURRENT_BINARY_DIR}/cub")
+    if (NOT CONDA_ROOT)
+        get_third_party(cub)
+        set(CUB_ROOT_DIR "${CMAKE_CURRENT_BINARY_DIR}/cub")
+    endif()
```

Comment:
Are you not relying on a pretty recent CUDA here? I honestly do not remember the version that came with CUB bundled, but it was likely 10 or even 11.

Reply:
Yeah, this conditional can probably get dropped, I was pretty conservative initially with preserving the cmake recipe as it was when conda wasn't specified.